pkg/solana/config/config.go

```diff
@@ -17,7 +17,8 @@ var defaultConfigSet = Chain{
 	OCR2CacheTTL:        config.MustNewDuration(time.Minute),            // stale cache deadline
 	TxTimeout:           config.MustNewDuration(time.Minute),            // timeout for send tx method in client
 	TxRetryTimeout:      config.MustNewDuration(10 * time.Second),       // duration for tx rebroadcasting to RPC node
-	TxConfirmTimeout:    config.MustNewDuration(30 * time.Second),       // duration before discarding tx as unconfirmed
+	TxConfirmTimeout:    config.MustNewDuration(30 * time.Second),       // duration before discarding tx as unconfirmed. Set to 0 to disable discarding tx.
```

Comment:
- What would be the use case of disabling discarding a tx? 
- Could we have a case where we stack confirmed transactions that don't go through and we are unable to drop them? Could this cause a memory leak and should be worry about it?